README.md

```diff
@@ -1,14 +1,20 @@
 # DeepSea
 A collection of Salt files for deploying, managing and automating Ceph.
 
-These goal is to manage multiple Ceph clusters with a single salt master.  At this time, only one Ceph cluster is managed.
+The goal is to manage multiple Ceph clusters with a single salt master.  At this time, only a single Ceph cluster can be managed.
 
 The [diagram](deepsea.png) should explain the intended flow for the orchestration runners and related salt states.
 
 ## Status
 Automatic discovery, configuration and deployment of Ceph clusters works. RGW
 deployment is currently broken. MDS deployment and CephFS creation works.
 
+## Support
+To learn more about DeepSea, take a look at the [Wiki](https://github.com/SUSE/DeepSea/wiki).
+
+There is also a dedicated mailing list [deepsea-users](http://lists.suse.com/mailman/listinfo/deepsea-users).
+If you have any questions, suggestions for improvements or any other
+feedback, please joins us there! We look forward to your contributions.
```

Comment:
We could add a note on issues, currently github itself, https://github.com/SUSE/DeepSea/issues/new too